models/shipping-method/src/shipping-rate/generator.ts

```diff
@@ -0,0 +1,16 @@
+import { Money } from '@commercetools-test-data/commons';
+import { Generator, fake } from '@commercetools-test-data/core';
+import { TShippingRate } from './types';
+
+// https://docs.commercetools.com/api/projects/shippingMethods#shippingrate
+
+const generator = Generator<TShippingRate>({
+  fields: {
+    price: fake(() => Money.random()),
+    freeAbove: fake(() => Money.random()),
```

Comment:
The documentation requests `TypedMoney` rather than `Money` for these fields, which in our case is going to be `CentPrecisionMoney`.

![image](https://github.com/commercetools/test-data/assets/8136478/633a4140-ac09-494c-ae1f-6518ec74180a)